lib/hawkular/inventory/inventory_api.rb

```diff
@@ -331,9 +337,22 @@ def create_resource_type(feed_id, type_id, type_name)
     # @param [String] resource_id Id of the resource
     # @param [String] resource_name Name of the resource
     # @param [Hash<String,Object>] properties Additional properties. Those are not the config-properties
-    # TODO allow to create this as child of another resource
     def create_resource(feed_id, type_path, resource_id, resource_name = nil, properties = {})
-      the_feed = hawk_escape feed_id
+      create_resource_under_resource(feed_id, type_path, [], resource_id, resource_name, properties)
+    end
+
+    # Create a resource of a given type under a given feed. To retrieve that resource
+    # you need to call {#get_resource}
+    # @param [String] feed_id Id of the feed to add the resource to
+    # @param [String] type_path Path of the resource type of this resource
+    # @param [Array<String>] parent_resource_ids Ids of the resources under which we create this resource
```

Comment:
Same as above - instead of requiring an Array of "sub-ids", we should probably ask for the canonical path of the (to be) parent and do the computation internally if needed.